### ISSUE TITLE
iOS feature request: AutoOpen support

### ISSUE THREAD

**Issue by wkornewald** [strongbox-password-safe/Strongbox#466]
Could you please add support for AutoOpen? This is how KeePassXC does it: https://keepassxc.org/docs/KeePassXC_UserGuide.html#_automatic_database_opening

**Comment by strongbox-mark** [comment]
Is this different from the "Quick Launch" feature? Do you use that?

Are you looking to open multiple different databases by opening 1?

**Comment by wkornewald** [comment]
Yes, I'd like to open multiple or at least one other DB from the first one.

**Comment by strongbox-mark** [comment]
OK, but how would you expect this to work visually, like how would you switch between databases on a small iOS screen?

**Comment by wkornewald** [comment]
Maybe the entries in the AutoOpen folder could get an „open“ button and then you’d navigate into the DB? The back button would bring you back to the outer DB.

Surely there are better UIs, but this would solve my primary use case easily. Alternatively a dropdown/menu/sidebar or something could allow switching DBs.

**Comment by strongbox-mark** [comment]
Isn't just popping back to your database list and opening whatever db you want just as convenient here?

**Comment by wkornewald** [comment]
Then I have to copy-paste the password and select the correct DB. That’s several extra steps compared to just selecting the DB.

**Comment by strongbox-mark** [comment]
I see. Is this the same feature as #401 ?

**Comment by wkornewald** [comment]
Not quite. I want to be able to open one DB via password and use that as the gateway into the other DBs without setting up biometry or storing the passwords for multiple DBs. The use case of for e.g. DBs shared with other people or different devices having access to different „security levels“ of DBs.

**Comment by wkornewald** [comment]
In my case they’re local. Like explained in the docs that I linked, the url field need to be set:

> You can setup one or more databases to open automatically when you unlock a single database. This is done by (1) defining a special group named AutoOpen with (2) entries that contain the file path and credentials for each database that should be opened. There is no limit to the number of databases that can be opened.

> Case matters with auto open, the group name must be exactly AutoOpen.

> Enter the absolute or relative path to the database file in the url field. You can also use the {DB_DIR} placeholder to reference the absolute path of the current database file.

> You can setup an entry to open on double click of the URL field by prepending kdbx:// to the relative or absolute path to the database file. You may also have to add file:// to access network shares (e.g., kdbx://file://share/database.kdbx).

**Comment by strongbox-mark** [comment]
OK, thanks for the feature request and the extra detail. Unfortunately I think this would fall down the priority list, and it's probably worth just using a convenience unlock method to make your life easier swapping between multiple databases.

**Comment by mfornet** [comment]
Same as #401
